### PR TITLE
OCLOMRS: Modify ReadMe

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
# JIRA TICKET NAME:
[OCLOMRS-11: Modify ReadMe](https://issues.openmrs.org/browse/OCLOMRS-11)

# Summary:
This PR adds the development steps in the ReadMe, tells the user how to set up the project inorder to get it working locally in their browsers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,16 @@
     $ npm install(if using npm)
     $ yarn install (if using yarn)
 ```
+### Run in development
+```
+- After installing all the dependecies then type the:
```
Comment (mwaz) [inline README.md]: I think the above name should be dependencies.

**Comment by hadijahkyampeire** [comment]
@EleisonC if the 3000 port is busy it just prompts you to type Y and the port is changed automatically. Thats what I meant

**Comment by EleisonC** [comment]
I know that but imagine someone else doesn't know that or if the something else is actually wrong with the port. So that is why I was thinking if you could probably add a way to change a port

**Comment by hadijahkyampeire** [comment]
@EleisonC I have added them

**Pull request merged**